app/controllers/agendas.js

```diff
@@ -205,19 +197,16 @@ export default class AgendasController extends Controller {
 
     // default decision result when flag is turned off
     let decisionResultCode = null;
-    if (!this.enableDigitalAgenda) {
-      decisionResultCode = await this.store.findRecordByUri(
-        'concept',
-        CONSTANTS.DECISION_RESULT_CODE_URIS.GOEDGEKEURD
-      );
-    }
+    decisionResultCode = await this.store.findRecordByUri(
+      'concept',
+      CONSTANTS.DECISION_RESULT_CODE_URIS.GOEDGEKEURD
+    );
```

Comment:
this should go, the flag was here to prevent the decisionResult from being set. (with a fallback to previous behavior if flag was off)

Reply:
fixed